### DISCUSSION TITLE
Can't log in to AciveCaptain

### DISCUSSION THREAD

**Discussion by rgregg** [sbender9/WilhelmSKSupport#33] [General]
I'm trying to log into ActiveCaptain on my iPhone, and it's not working. I can successfully authenticate, but then I see the following on the screen:

```
{serviceUrl: 'https://sso.garmin.com/sso/embed', serviceTicket: 'ST-0246000-<random digits>'}
```

Is the integration broken right now?

**Comment by sbender9** [comment]
Yeah. Sorry. It's broken. They changed something. On my list to figure out. But pretty low on the list at this point. 

The only reason to login is to make updates/comments/etc.

**Comment by rgregg** [comment]
Thanks. I was trying to figure out how to get the Navionics charts to display, is there another way to do that?

**Reply by sbender9** [reply]
Just enable "Show On Maps". Don't tell me that doesn't work now?

**Reply by sbender9** [reply]
Yep. Broke.
